applications/FluidDynamicsApplication/fluid_dynamics_application.h

```diff
@@ -82,6 +82,8 @@
 #include "custom_elements/compressible_navier_stokes_explicit.h"
 #include "custom_elements/two_fluid_navier_stokes.h"
 #include "custom_elements/two_fluid_navier_stokes_alpha_method.h"
+#include "custom_elements/two_fluid_navier_stokes_fractional.h"
+#include "custom_elements/vectorial_convection_fractional_element.h"
```

Comment:
I think that the name of this file, and the class, should indicate that it is understood to work together to the two_fluid_navier_stokes_fractional. Consider renaming it to something like `two_fluid_navier_stokes_fractional_convection.h` or something of this sort.